scheduler/generic_sched_test.go

```diff
@@ -6987,9 +6961,9 @@ func TestDowngradedJobForPlacement_PicksTheLatest(t *testing.T) {
 		// update is a "destructive" update and has been updated manually
 		promoted bool
 
-		// requireCanaries indicate whether the job update requires placing canaries due to
+		// mustCanaries indicate whether the job update musts placing canaries due to
```

Comment:
"mustCanaries" is probably fine for the variable though

```suggestion
		// mustCanaries indicate whether the job update requires placing canaries due to
```